gcp-build-server-aos.cfg

```diff
@@ -11,3 +11,5 @@ workspace_cache_base_dir = /mnt/work/tmp
 # guests domains which to be built and installed
 XT_GUESTS_INSTALL = "domf"
 XT_GUESTS_BUILD = "domf"
+# aos specific local.conf configuration
+BBMASK = "../meta-aos/recipes-aos"
```

Comment:
frankly, i am not sure if it will work always.
Because, the BBMASK can be defined somewhere in the sample of local.conf and I am afraid, the BBMASK will be overwritten. For example, the local.conf.rcar-domu-image-weston has the following code BBMASK = "meta-linaro/meta-optee/recipes-security/optee". The proposed approach will not work here. If there is a conflict between the recipes, maybe it is reasonable to use the PREFERRED_PROVIDER_.
Please, describe the motivation of the change, to find really right decision